kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -0,0 +1,162 @@
+import { parseParcelPosition } from '../atomicHelpers/parcelScenePositions'
+import { UnityPortableExperienceScene } from './UnityParcelScene'
+import {
+  forceStopParcelSceneWorker,
+  getParcelSceneID,
+  getSceneWorkerBySceneID,
+  loadParcelScene
+} from 'shared/world/parcelSceneManager'
+import { unityInterface } from './UnityInterface'
+import { parseUrn, DecentralandAssetIdentifier, OffChainAsset } from '@dcl/urn-resolver'
+
+const STATIC_PORTABLE_SCENES_S3_BUCKET_URL = 'https://static-pe.decentraland.io'
+export type PortableExperienceUrn = string
+
+declare var window: any
+window['spawnPortableExperienceScene'] = spawnPortableExperienceScene
+window['killPortableExperienceScene'] = killPortableExperienceScene
+
+export type PortableExperienceIdentifier = string
+export type PortableExperienceHandle = {
+  pid: PortableExperienceIdentifier
+  parentCid: string
+}
+
+let currentPortableExperiences: Map<string, string> = new Map()
+
+export function getPortableExperience(pid: string): PortableExperienceHandle | undefined {
+  if (currentPortableExperiences.has(pid)) {
+    return { pid: pid, parentCid: currentPortableExperiences.get(pid)! }
+  } else {
+    return undefined
+  }
+}
+
+export async function spawnPortableExperienceScene(
+  portableExperienceUrn: PortableExperienceUrn,
+  parentCid: string
+): Promise<PortableExperienceHandle> {
+  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(portableExperienceUrn)
+
+  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
+    throw new Error(`Could not parse portable experience from urn: ${portableExperienceUrn}`)
+  }
+
+  /* tslint:disable-next-line */
+  const scene = new UnityPortableExperienceScene(await getPortableExperienceFromS3Bucket(parsedUrn as OffChainAsset))
+  loadParcelScene(scene, undefined, true)
+  const parcelSceneId = getParcelSceneID(scene)
+  unityInterface.CreateUIScene({
+    id: parcelSceneId,
+    name: scene.data.name,
+    baseUrl: scene.data.baseUrl,
+    contents: scene.data.data.contents,
+    icon: scene.data.data.icon,
+    isPortableExperience: true
+  })
+
+  currentPortableExperiences.set(parcelSceneId, parentCid)
+
+  return { pid: parcelSceneId, parentCid: parentCid }
+}
+
+function isPortableExperience(dclId: DecentralandAssetIdentifier): dclId is OffChainAsset {
+  if (dclId) {
+    /* tslint:disable-next-line */
+    return !!(dclId as OffChainAsset).registry && (dclId as OffChainAsset).registry === 'static-portable-experiences'
+  }
+  return false
+}
+
+export async function killPortableExperienceScene(portableExperienceUrn: PortableExperienceUrn): Promise<boolean> {
+  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(portableExperienceUrn)
+  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
+    throw new Error(`Could not parse portable experience from urn: ${portableExperienceUrn}`)
+  }
+  const sceneId: string = parsedUrn.id
+  const peWorker = getSceneWorkerBySceneID(sceneId)
```

Comment:
👾 The `sceneId` of the portable experience should be the whole URN. It will also prevent duplicates and it will prevent us from doing `urn:decentraland:off-chain:static-portable-experiences:${data.portableExperienceId}` everywhere.